src/scripts/site.js

```diff
@@ -745,19 +731,18 @@ var AppInfo = {};
         define("libjass", [bowerPath + "/libjass/libjass.min", "css!" + bowerPath + "/libjass/libjass"], returnFirstDependency);
         define("tunerPicker", ["components/tunerpicker"], returnFirstDependency);
         define("mainTabsManager", [componentsPath + "/maintabsmanager"], returnFirstDependency);
-        define("imageLoader", [componentsPath + "/images/imagehelper"], returnFirstDependency);
+        define("imageLoader", [componentsPath + "/images/imageLoader"], returnFirstDependency);
         define("appFooter", [componentsPath + "/appfooter/appfooter"], returnFirstDependency);
         define("directorybrowser", ["components/directorybrowser/directorybrowser"], returnFirstDependency);
         define("metadataEditor", [componentsPath + "/metadataeditor/metadataeditor"], returnFirstDependency);
         define("personEditor", [componentsPath + "/metadataeditor/personeditor"], returnFirstDependency);
-        define("playerSelectionMenu", [componentsPath + "/playback/playerselection"], returnFirstDependency);
+        define("playerSelectionMenu", [componentsPath + "/playback/playerSelectionMenu"], returnFirstDependency);
         define("playerSettingsMenu", [componentsPath + "/playback/playersettingsmenu"], returnFirstDependency);
         define("playMethodHelper", [componentsPath + "/playback/playmethodhelper"], returnFirstDependency);
         define("brightnessOsd", [componentsPath + "/playback/brightnessosd"], returnFirstDependency);
         define("libraryMenu", ["scripts/librarymenu"], returnFirstDependency);
         define("emby-collapse", [componentsPath + "/emby-collapse/emby-collapse"], returnFirstDependency);
         define("emby-button", [componentsPath + "/emby-button/emby-button"], returnFirstDependency);
-        define("emby-linkbutton", ["emby-button"], returnFirstDependency);
```

Comment:
haha someone got lazy

Reply:
this type of changes are a nightmare as you can notice